analytics.js

```diff
@@ -68,6 +80,25 @@ function pageview(url) {
 }
 
 /**
+ * Set a custom dimension for future page views and events
+ * @param name
+ * @param cookieId
+ */
+function dimensionByCookie(name, cookieId) {
```

Comment:
@DFurnes How would you feel about keeping this function, but making a new one just called `dimension` and exposing both. I think it'll help keep things a bit dry across the various platforms